opal/include/opal/sys/arm64/atomic.h

```diff
@@ -169,7 +169,7 @@ static inline bool opal_atomic_compare_exchange_strong_acq_64(opal_atomic_int64_
     __asm__ __volatile__("1:  ldaxr   %0, [%2]        \n"
                          "    cmp     %0, %3          \n"
                          "    bne     2f              \n"
-                         "    stxr    %w1, %4, [%2]   \n"
+                         "    stlxr   %w1, %4, [%2]   \n"
```

Comment:
unnecessary release.

Reply:
Yes, you are right,  I updated the modification only in  exchange_strong_32 and exchange_strong_64